code/espurna/scheduler.cpp

```diff
@@ -62,7 +71,7 @@ void _schWebSocketOnConnected(JsonObject &root){
         switch_.add(getSetting({"schSwitch", i}, 0));
         action.add(getSetting({"schAction", i}, 0));
         type.add(getSetting({"schType", i}, SCHEDULER_TYPE_SWITCH));
-        hour.add(getSetting({"schHour", i}, 0));
+        hour.add(getSetting({"schHour", i}, 12));
```

Comment:
The default value in the custom.js UI is set to 12h, if not edited, the final stored value will be 0h even if you use save button. After reboot it will be 0h

Reply:
Instead of changing default, we should change UI to display the correct value.
This looks like bug from #1986, 12 should not have been there